CommandLine/testing/Tests/tiles_room_transition_test.gmx/help.rtf

```diff
@@ -0,0 +1,7 @@
+{\rtf1\ansi
```

Comment:
You can probably delete this file and the icon without damaging anything. At least, I'd hope you can.